src/Perspex.Controls/Window.cs

```diff
@@ -44,6 +47,22 @@ public enum SizeToContent
     /// </summary>
     public class Window : TopLevel, IStyleable, IFocusScope, ILayoutRoot, INameScope
     {
+        private static IList<Window> windows = new List<Window>();
```

Comment:
Static members should start with the `s_` prefix according to the .net core coding style that we follow.